results/test-results.xml

```diff
@@ -0,0 +1,255 @@
+    </testcase>
+    <testcase name="Contract: RRC20 Approvals: msg.sender should approve 100 to accounts[1]" time="0.0420" classname="Approvals: msg.sender should approve 100 to accounts[1]">
+    </testcase>
+    <testcase name="Contract: RRC20 Approvals: msg.sender approves accounts[1] of 100 &amp; withdraws 20 once." time="0.0640" classname="Approvals: msg.sender approves accounts[1] of 100 &amp; withdraws 20 once.">
+    </testcase>
+    <testcase name="Contract: RRC20 Approvals: msg.sender approves accounts[1] of 100 &amp; withdraws 20 twice." time="0.1190" classname="Approvals: msg.sender approves accounts[1] of 100 &amp; withdraws 20 twice.">
+    </testcase>
+    <testcase name="Contract: RRC20 Approvals: msg.sender approves accounts[1] of 100 &amp; withdraws 50 &amp; 60 (2nd tx should fail)" time="0.0840" classname="Approvals: msg.sender approves accounts[1] of 100 &amp; withdraws 50 &amp; 60 (2nd tx should fail)">
+    </testcase>
+    <testcase name="Contract: RRC20 Approvals: Attempt withdrawal from account with no allowance (should fail)" time="0.0050" classname="Approvals: Attempt withdrawal from account with no allowance (should fail)">
+    </testcase>
+    <testcase name="Contract: RRC20 Approvals: Allow accounts[1] 100 to withdraw from accounts[0]. Withdraw 60 and then approve 0 &amp; attempt transfer." time="0.1110" classname="Approvals: Allow accounts[1] 100 to withdraw from accounts[0]. Withdraw 60 and then approve 0 &amp; attempt transfer.">
+    </testcase>
+    <testcase name="Contract: RRC20 Approvals: Approve max (2^256 - 1)" time="0.0650" classname="Approvals: Approve max (2^256 - 1)">
+    </testcase>
+    <testcase name="Contract: RRC20 Approvals: msg.sender approves accounts[1] of max (2^256 - 1) &amp; withdraws 20" time="0.0580" classname="Approvals: msg.sender approves accounts[1] of max (2^256 - 1) &amp; withdraws 20">
+    </testcase>
+    <testcase name="Contract: RRC20 Events: Should fire Transfer event properly" time="0.0280" classname="Events: Should fire Transfer event properly">
+    </testcase>
+    <testcase name="Contract: RRC20 Events: Should fire Transfer event normally on a zero transfer" time="0.0540" classname="Events: Should fire Transfer event normally on a zero transfer">
+    </testcase>
+    <testcase name="Contract: RRC20 Events: Should fire Approval event properly" time="0.0190" classname="Events: Should fire Approval event properly">
+    </testcase>
+  </testsuite>
+  <testsuite name="Contract: RRC20Factory" timestamp="2021-09-20T14:05:52" tests="1" file="/Users/jjtechera/Documents/GitHub/rskj-truffle-tests/test/token/rrc20/rrc20Factory.js" time="0.1270" failures="0">
+    <testcase name="Contract: RRC20Factory Verify a Human Standard Token once deployed using both verification functions." time="0.1270" classname="Verify a Human Standard Token once deployed using both verification functions.">
+    </testcase>
+  </testsuite>
+</testsuites>
```

Comment:
`test-results.xml` should not be uploaded to the repo.

Reply:
You are right. Sorry for this.